src/main.roc

```diff
@@ -1,6 +1,6 @@
 platform "cli"
     requires {} { main : Task {} [] }
-    exposes [Task, Process, Stdout, Stderr, Stdin, Path, File, FileMetadata, Dir, Arg, Env, EnvDecoding, Url, Http]
+    exposes []
```

Comment:
@rtfeldman this fixes the compiler panics that occur when building env.roc and file.roc, but it may cause no docs to be generated? Also strangely, even though nothing is exposed, `Task, Stdout, Stderr` can still be used by for example `env.roc`.

Reply:
This also prevents the false positive import error from occurring that I talked to you about earlier:
```
── MODULE NOT IMPORTED ──── ../../basic-cli2/basic-cli/examples/../src/Env.roc ─

The `EnvDecoding` module is not imported:

78│                          Decode.fromBytes (Str.toUtf8 varStr) (EnvDecoding.format {})
                                                                   ^^^^^^^^^^^^^^^^^^

Did you mean to import it?

────────────────────────────────────────────────────────────────────────────────
```
Which was caused by commit [e549456](https://github.com/roc-lang/roc/pull/4652/commits/e5494566689b62f2ca509a1d58186199dceedf51)